### PR TITLE
WEBUI-706: prevent vocabulary creation with whitespaces only in id field - 10.10

### PR DESCRIPTION
**Pull request by richardsd**
Backport of https://github.com/nuxeo/nuxeo-web-ui/pull/1405

Cross repo check https://github.com/nuxeo/nuxeo-web-ui/actions/runs/1928797443 🔵

### CODE REVIEW

**Comment by richardsd** [comment]
WEBUI-706-10 10

File: i18n/messages.json [hunk 1]
```diff
@@ -1483,6 +1483,7 @@
   "vocabularyManagement.confirmDelete": "Delete vocabulary entry?",
   "vocabularyManagement.deleteEntry": "Delete Entry",
   "vocabularyManagement.edit.actions": "Actions",
+  "vocabularyManagement.edit.error": "ID cannot be empty",
```
Comment (Gabez0r) [inline i18n/messages.json:1486]: In theory, because we are adding a new label, we need to do the translations first and then backport them to this PR.

Comment (richardsd) [reply i18n/messages.json:1486]: Ah, you mean we should cherry-pick all the translations from the PR for `maintenance-3.0.x` to this one?

**Review by semisse — DISMISSED** [review]

File: i18n/messages.json (hunk 1 above)
Comment (richardsd) [reply i18n/messages.json:1486]: Waiting for https://jira.nuxeo.com/browse/NXDOC-2436 to cherry-pick the translations and update the PR. Thanks for the review Gabe.

Comment (richardsd) [reply i18n/messages.json:1486]: Labels are updated, thanks for the review.

**Comment by richardsd** [comment]
@Gabez0r @semisse asking for another round of reviews because I backported the translations from `maintenance-3.0.x`. Thank you.

**Review by Gabez0r — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Pull request merged**